WordPress/src/main/java/org/wordpress/android/ui/media/MediaItemFragment.java

```diff
@@ -211,7 +214,12 @@ private void refreshViews(MediaModel mediaModel) {
             mDescriptionView.setVisibility(View.VISIBLE);
         }
 
-        mDateView.setText(mediaModel.getUploadDate());
+        Date date = DateTimeUtils.dateFromIso8601(mediaModel.getUploadDate());
```

Comment:
Let's add a null check before passing `mediaModel.getUploadDate()` to `dateFromIso8601`. Otherwise a NPE can happen here (note the `dateFromIso8601` only catches a `ParseException` type).